irohad/torii/impl/command_service_impl.cpp

```diff
@@ -129,8 +129,29 @@ namespace iroha {
       using ResponsePtrType =
           std::shared_ptr<shared_model::interface::TransactionResponse>;
       auto initial_status = cache_->findItem(hash).value_or([&] {
-        log_->debug("tx is not received: {}", hash);
-        return status_factory_->makeNotReceived(hash);
+        // if cache_ didn't contain some status there is required to check
+        // persistent cache
+
+        log_->debug("tx {} don't present in cache", hash);
```

Comment:
Doesn't

Reply:
**Isn't** since it is a form of "to be present" but not an active verb "to present".